app/controllers/step_by_step_pages_controller.rb

```diff
@@ -101,6 +101,16 @@ def unpublish
     end
   end
 
+  def unschedule
```

Comment:
For safety and consistency, this action should be restricted to users who have the "Scheduling" permission.